src/Aquifer.API/Endpoints/Projects/Create/Endpoint.cs

```diff
@@ -169,6 +176,75 @@ private async Task<List<ResourceContentEntity>> CreateOrFindAquiferizationResour
         return resourceContents;
     }
 
+    private async Task<List<ResourceContentEntity>> CreateOrFindTranslatedResourceContentAsync(
```

Comment:
This method could benefit from detailed review to make sure I got the validation and creation correct.